src/azure-cli-testsdk/azure/cli/testsdk/scenario_tests/tests/test_recording_processor.py

```diff
@@ -107,11 +107,14 @@ def test_subscription_recording_processor_for_response(self):
 
         for template in body_templates:
             mock_sub_id = str(uuid.uuid4())
-            mock_response = dict({'body': {}})
```

Comment:
This `dict()` is a no-op, not sure why it was written like that in the first place.